src/main/java/seedu/address/logic/Messages.java

```diff
@@ -14,7 +14,7 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person name provided is invalid";
```

Comment:
Isnt this should be PERSON_DISPLAYED_NAME to avoid confusion?

Reply:
yea I saw I should've changed I'll change in the next iteration!